code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -107,6 +107,7 @@
 /datum/design/unique_modkit
 	category = list("Mining Designs", "Cyborg Upgrade Modules") //can't be normally obtained
```

Comment:
So does adding the dep flag add to always being available at the lathe? Because the intent with these is that you need the disk to unlock the tech, as opposed to being able to make it right off the bat.

Reply:
nah it just allows them to be printed once unlocked.